src/TimeGrid.jsx

```diff
@@ -277,6 +294,15 @@ export default class TimeGrid extends Component {
     }
   }
 
+  adjustScroll() {
+    const { min, max, scrollTime } = this.props;
+
+    const diffMillis = scrollTime - dates.startOf(scrollTime, 'day');
```

Comment:
what if ScrollTime is undefined? probably should return early if so to avoid extra work or possible null exceptions